MailCore/Cache/AccountManager/AccountManager.swift

```diff
@@ -240,6 +243,7 @@ public final class AccountManager: RefreshTokenDelegate, ObservableObject {
 
         let apiFetcher = getApiFetcher(for: account.userId, token: token)
         let user = try await userProfileStore.updateUserProfile(with: apiFetcher)
+        try await myKSuiteStore.updateMyKSuite(with: apiFetcher, id: user.id)
```

Comment:
This shouldn't break user login
```suggestion
        try? await myKSuiteStore.updateMyKSuite(with: apiFetcher, id: user.id)
```